Latency_Extractor_ReadMe.md

```diff
@@ -0,0 +1,49 @@
+
+# Kuksa Data Broker
```

Comment:
If we are to merge this PR I would like a link from the main README to this file